discord_lib/discord_chat_service.py

```diff
@@ -46,11 +48,36 @@ def delete_channel(self, channel_id):
         if int(channel_id) in channels_by_id:
             self._client.channels_delete(int(channel_id))
 
+    def get_or_create_category(self, category_name):
+        """
+        Returns category that has fewer than 50 children channels. If none
```

Comment:
nit: consider replacing "50 children channels" with "max number of children channels specified in the constructor" or something like that, since it is parameterized

Reply:
Done.